sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -118,12 +118,22 @@ export class BatchingReceiver extends MessageReceiver {
         this.name
       );
 
-      return await this._batchingReceiverLite.receiveMessages({
+      const messages = await this._batchingReceiverLite.receiveMessages({
         maxMessageCount,
         maxWaitTimeInMs,
         maxTimeAfterFirstMessageInMs,
         userAbortSignal
       });
+
+      if (this._lockRenewer) {
+        for (const message of messages) {
+          this._lockRenewer.start(this, message, (error) => {
+            logError(error, `${this.logPrefix} Failed to renew lock for message.`);
```

Comment:
Should we perhaps log the message or message id here? Something for the user to connect the failure to the message?

Reply:
message-id, please!

Reply:
I'm going to remove it - we actually have this rather nice message in the autolockrenewer today already:
                
logError(
                  err,
                  `${logPrefix} An error occurred while auto renewing the message lock '${bMessage.lockToken}' for message with id '${bMessage.messageId}'`
                );